src/settings/LegalScreen.js

```diff
@@ -15,6 +15,7 @@ type Props = $ReadOnly<{|
   route: RouteProp<'legal', void>,
 |}>;
 
+/** (NB this is a per-account screen: it leads to this realm's policies.) */
```

Comment:
Indeed. Not even a bug, really -- "LegalScreen" is accurate (if vague), and doesn't say anything wrong. It's just that one screen you could imagine us having (because lots of apps do) is one that has legal terms related to the app itself, and that would naturally be part of the global non-per-account UI.

I'll reword.